crates/uv-cli/src/lib.rs

```diff
@@ -1980,6 +1980,10 @@ pub struct SyncArgs {
     /// - `/home/ferris/.local/bin/python3.10` uses the exact Python at the given path.
     #[arg(long, short, env = "UV_PYTHON", verbatim_doc_comment)]
     pub python: Option<String>,
+
+    /// The path to the project. Defaults to the current working directory.
+    #[arg(long, hide = true)]
+    pub directory: Option<PathBuf>,
```

Comment:
I need this for the benchmarking script (we can't change directories because we use a single hyperfine command, and that has to run from a consistent directory), but we can discuss whether it should be exposed to users. (I think it should... Very useful.)